### DISCUSSION TITLE
Reviewing board for introduction of new rules

### DISCUSSION THREAD

**Discussion by ssbarnea** [ansible/ansible-lint#1608] [New rules]
Once a new rule is proposed with its own discussion thread and reaches a mature form, it should be announced here with a link to it. Each new rule link should be posted as a different answer.

This assures that those interested in reviewing introduction of new rules have the opportunity to give feedback before the new rule is introduced. They can just subscript to this particular discussion.

**Comment by gundalow** [comment]
Great idea, nice use of GitHub Discussions

**Comment by ssbarnea** [comment]
https://github.com/ansible-community/ansible-lint/pull/1614 introduce an opt-in rule that assures that all build-ins are called using FQCN names.
